pkg/model/employees.go

```diff
@@ -24,47 +24,51 @@ type Employee struct {
 	BaseModel
 
 	// basic info
-	FullName                string
-	DisplayName             string
-	TeamEmail               string
-	PersonalEmail           string
-	Avatar                  string
-	PhoneNumber             string
-	Address                 string
-	MBTI                    string
-	Gender                  string
-	Horoscope               string
-	PassportPhotoFront      string
-	PassportPhotoBehind     string
-	IdentityCardPhotoFront  string
-	IdentityCardPhotoBehind string
-	DateOfBirth             *time.Time
+	FullName               string     `gorm:"column:full_name"`
```

Comment:
I use gorm to match with the table in the database because of in some cases like BasecampAttachableSGID string `gorm:"column:basecamp_attachable_sgid"` the database cant match with the right column or I have to change the name to BasecampAttachableSgid as I did before

Reply:
json has the same effect gorm

Reply:
Yeah. That's right. Try to use JSON tags for more benefits. We only use gorm tags for the specific case eg define the relation to another entity.